src/cmd/barriers/barriers.cpp

```diff
@@ -38,6 +38,13 @@ main(int argc, char** argv)
     // record window instance for tray icon, etc
     ArchMiscWindows::setInstanceWin32(GetModuleHandle(NULL));
 #endif
+
+#ifdef __APPLE__
+	/* Silence "is calling TIS/TSM in non-main thread environment" as it is a red
```

Comment:
Can we use spaces here? Most of the surrounding code does that.

Reply:
Yeah, I'll fix that as soon as I get home, sorry about that

Reply:
@p12tic Not sure how those pesky tabs got there but they're gone now